frontend/src/components/common/Header/Cart/cartSlice.ts

```diff
@@ -0,0 +1,52 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { CartProps } from './Cart';
+
+export interface CartState {
+  cartList: CartProps[];
+  total: number;
+}
+
+const initialState: CartState = {
+  cartList: [],
+  total: 0,
+};
+const cartSlice = createSlice({
+  name: 'cart',
```

Comment:
You should define a const for all slice names, don't use string for sth like name, id

Reply:
@thuanvuong-kms  I don't clear about this point. Should I define a const value  for that string in src/constants/slice and use it ?

Reply:
yes, you can create a const like this SLICE_NAMES = { CART: 'cart', ... }